### ISSUE TITLE
Thicker edges in UI when a count property is larger

### ISSUE THREAD

**Issue by p013570** [gchq/gaffer-tools#740]
If there is a a "count" property defined in the UI config, then use this to determine how thick to draw the edges in the graph UI.

**Comment by p013570** [comment]
Just needs tests writing.

**Comment by n3101** [comment]
The UI fails to meet our Accessibility policy. It is unmaintainable and will be removed shortly.
